Documentation/PageTsconfig/Mod/_WebList/_downloadPresets.tsconfig

```diff
@@ -0,0 +1,14 @@
+mod.web_list.downloadPresets {
+    pages {
+        10 {
```

Comment:
Locally I have tested that with `minimum` and `fullExport`. Would be much easier to remember and to overwrite if we give these TypoScript paths a better name.

Reply:
So you tested and non-numeric keys are possible?

Reply:
Yes, it's just normal typoscript "path building", so numeric or string is allowed

Reply:
Done

Reply:
Yes, tested with page TSconfig directly on a page on a working GIT version of TYPO3.